packages/playground/website/playwright/e2e/website-helpers.ts

```diff
@@ -0,0 +1,81 @@
+import { expect, Page } from '../playground-fixtures.ts';
+
+export const expandSiteView = async (page: Page) => {
+	const openSiteButton = page.locator('div[title="Open site"]');
+	await expect(openSiteButton).toBeVisible();
+	await openSiteButton.click();
```

Comment:
I'd discourage using assertions within utilities to separate concerns (action vs verification). The good part is that dropping those assertions won't make those utilities more fragile. It's because the UI actions API (click, fill, check, focus, etc.) [implies element visibility](https://playwright.dev/docs/actionability#introduction), so they internally check if the element is visible before performing the actual action.

Reply:
Makes sense, done.